src/components/container/Profile/Profile.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Header from '@components/commons/Header';
+import {fetchGuest} from '@actions/profile';
+import Footer from '@components/commons/utilities/Footer';
+import ProfileImage from './ProfileImage';
+import ProfileDisplay from './ProfileDisplay';
+import './Profile.scss';
+
+class Profile extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tabs: [ 'Followers', 'Articles', 'Bookmarks' ],
+      clickedTab: 'Articles',
+      isMyProfile: false
+    }
+  }
+
+  componentDidMount() {
+    const { user: {username}, fetchGuest, history  } = this.props;
+    const { match: { params: { username: usernameURL } } } = this.props;
+    if (username === usernameURL) {
```

Comment:
You could probably do this in the constructor.

`isMyProfile: this.props.match.params.username === this.props.user.username`

Reply:
this is already implemented. Thanks!